src/styles.scss

```diff
@@ -0,0 +1,364 @@
+$font: Lucida Sans Unicode;
+$lightblueBg: 'https://www.tutorialchip.com/wp-content/uploads/2011/06/Blue-Pattern.jpg';
+$brownButtonColor: linear-gradient(90deg, rgba(186,129,120,1) 0%, rgba(135,77,69,1) 100%);
+$brownButtonColorActive: linear-gradient(90deg, rgba(135,77,69,1) 0%, rgba(186,129,120,1) 100%);
+$scrollbarTrack: #559FB4;
+$scrollbarThumbHover: #a86257;
+$cardBg: 'https://i.pinimg.com/originals/dd/6d/c8/dd6dc8a6b6db1b6f4b01bd1bcc891621.jpg';
+
+//Gradients
+
+$red: linear-gradient(45deg, rgba(69,1,2,1) 0%, rgba(255,0,66,1) 50%, rgba(69,1,2,1) 100%);
+$lightred: linear-gradient(45deg, rgba(254,64,90,1) 0%, rgba(247,182,186,1) 50%, rgba(254,64,90,1) 100%);
+$blue: linear-gradient(45deg, rgba(47,55,153,1) 0%, rgba(2,245,251,1) 50%, rgba(47,55,153,1) 100%);
+$darkblue: linear-gradient(45deg, rgba(12,16,60,1) 0%, rgba(12,25,219,1) 50%, rgba(14,19,80,1) 100%);
+$lightblue: linear-gradient(45deg, rgba(99,116,208,1) 0%, rgba(208,237,255,1) 50%, rgba(99,116,208,1) 100%);
+$green: linear-gradient(45deg, rgba(10,53,8,1) 0%, rgba(78,243,79,1) 50%, rgba(10,53,8,1) 100%);
+$yellowBlue: linear-gradient(45deg, rgba(64,59,74,1) 5%, rgba(224,228,138,1) 50%, rgba(64,59,74,1) 95%);
+$mojito: linear-gradient(45deg, rgba(29,151,108,1) 0%, rgba(147,249,185,1) 50%, rgba(29,151,108,1) 100%);
+$pink: linear-gradient(45deg, rgba(219,24,182,1) 0%, rgba(255,200,252,1) 50%, rgba(219,24,182,1) 100%);
+$purple: linear-gradient(45deg, rgba(65,39,173,1) 0%, rgba(213,154,238,1) 50%, rgba(65,39,173,1) 100%);
+$yellow: linear-gradient(45deg, rgba(255,252,0,1) 0%, rgba(255,255,255,1) 50%, rgba(255,252,0,1) 100%);
+$brown: linear-gradient(45deg, rgba(51,40,27,1) 0%, rgba(156,100,41,1) 50%, rgba(51,40,27,1) 100%);
+$orange: linear-gradient(45deg, rgba(241,39,17,1) 0%, rgba(245,175,25,1) 50%, rgba(241,39,17,1) 100%);
+$lightgray: linear-gradient(45deg, rgba(108,107,113,1) 0%, rgba(207,207,207,1) 50%, rgba(108,107,113,1) 100%);
+$darkgray: linear-gradient(45deg, rgba(31,28,44,1) 0%, rgba(146,141,171,1) 50%, rgba(31,28,44,1) 100%);
+$peachy: linear-gradient(45deg, rgba(65,94,249,1) 0%, rgba(248,71,110,1) 50%, rgba(65,94,249,1) 100%);
+$darkPurple: linear-gradient(45deg, rgba(51,0,27,1) 0%, rgba(255,0,132,1) 50%, rgba(51,0,27,1) 100%);
+$martini: linear-gradient(45deg, rgba(36,254,65,1) 0%, rgba(253,252,71,1) 50%, rgba(36,254,65,1) 100%);
+
+body{
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  font-size: 18px;
+  font-family: $font;
+  background-image: url($lightblueBg);
+  background-size: contain;
+
+  .flex{
```

Comment:
are you appending `.flex` class to each flex component? I suggest to use mixins instead - the code become more readable, since we don't need to guess what component uses `.flex` class. and where.

https://sass-lang.com/documentation/at-rules/mixin

Reply:
Done